<launch>
    <!-- joy node -->
    <param name="realtime_reg_status_publish_rate" value="50" type="int"/>
    <param name="vel_limit_rpm" value="250" type="int"/>
    <param name="acc_lim_per_sec" value="2000" type="int"/>
    <param name="decel_lim_per_sec" value="800" type="int"/>
    <param name="torq_percantage" value="100" type="int"/>
    <param name="wheel_radius" value="0.23495" type="double"/> <!-- 9.25 inches -->
    <param name="wheel_radius_multiplier" value="1" type="double"/>
    <param name="wheel_base" value="0.54" type="double"/> <!-- 21.5 inches -->
    <param name="wheel_base_multiplier" value="1" type="double"/>
    <param name="vehicle_width" value="0.9144" type="double"/><!-- 36 inches -->
    <param name="vehicle_width_multiplier" value="1" type="double"/>
    <param name="gear_reduction" value="15" type="double"/>

    <node respawn="true" pkg="clearpath" type="client" name="servos_client" output="screen"/>
   

</launch>
